packages/server/lib/browsers/cri-client.ts

```diff
@@ -147,6 +147,10 @@ export const create = async (target: string, onAsynchronousError: Function, host
       })
 
       enqueuedCommands = []
+
+      if (onReconnect) {
```

Comment:
see [original(https://github.com/cypress-io/cypress/pull/18075/files#diff-4eee46e0819a421b758964abfcf2ea357773afceaff00ceafd16b89f193f4c4eR191) in `9.x`